infrastructure/helm-chart/charts/prerequisites/charts/kafka/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v1
 appVersion: "1.0"
 description: A kafka helm chart for Airy Platform
-name: airy
-version: 0.5.0
+name: kafka
+version: 2.7.0
```

Comment:
The version of the Kafka chart, should be in accordance with the version of Kafka.